### DISCUSSION TITLE
DNSSEC support

### DISCUSSION THREAD

**Discussion by q7nm** [shadowsocks/shadowsocks-rust#753] [Q&A] [answered]
I setup local-dns on 5353 port and redirected this to dnsmasq:
```
...
server=::1#5353
server=127.0.0.1#5353
...
```
But with DNSSEC it doesn't work:
```
...
conf-file=/usr/share/dnsmasq/trust-anchors.conf
dnssec
...
```

**Comment by zonyitoo** [comment, marked as answer]
The `local-dns` just provides plain TCP and UDP DNS query API. It would be too much work to do for implementing DoT, DoH, DNSSEC and I also don't think it is worth. The DNS service that provided by `local-dns` is just for clients that are on the same machine or in the same subnet.

But still, PRs are welcome. Because the DNS library we are using, the `trust-dns`, have already supports DoT, DoH, DNSSEC, which is a good start.
